### PR TITLE
Create issue template

### PR DESCRIPTION
**Pull request by Jguer**
#188

### CODE REVIEW

**Comment by Morganamilo** [comment]
I think what I said in #188 Is most of what I have to say on this.

I don't know how I feel about templates to be honest. They do force people to write better issues who otherwise wouldn't. But they get in the way of people who write good issues in the first place.

I tweaked the layout a little, too many comments maybe?

```markup
<!-- The following template is just a suggestion -->
<!-- Feel free to edit or discard it if you prefer a different format -->

#### Affected Version
<!-- Please ensure you are using the latest yay-git package -->
<!-- Use `yay -V` to get installed version -->
<!-- Example: `yay v3.373` -->

#### Expected Results
<!-- Example: Package installed successfully -->

#### Actual Results
<!-- Example: Error is thrown -->
<!-- Pasting terminal text is encouraged -->
<!-- If you think you know the cause feel free to explain>

###Steps to reproduce
<!-- Exact commands are useful -->
1.
2.
3.
```

**Comment by Jguer** [comment]
I like the just as a suggestion but it also reminds people of needed information, I think removing the Expected results and Actual results would be a better idea. What I really want is to know if they've tried the git version first. 

For example #185 I used it was a problem in master because it had been touched around but I couldn't be certain by the issue.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<!-- Example: `yay v3.373` -->
+
+#### Issue
+
+### Steps to reproduce
+
+<!-- Exact commands are useful -->
+
+1. 2. 3.
```
Comment (Morganamilo) [inline ISSUE_TEMPLATE.md]: Markup really doesn't like this, doing it on separate lines would be better.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+<!-- Please ensure you are using the latest yay-git package -->
+
+<!-- Use `yay -V` to get installed version -->
+
+<!-- Example: `yay v3.373` -->
+
+#### Issue
+
+### Steps to reproduce
```
Comment (Morganamilo) [inline ISSUE_TEMPLATE.md]: This is a different header size than the others.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!-- The following template is just a suggestion -->
+
+<!-- Feel free to edit or discard it if you prefer a different format -->
```
Comment (Morganamilo) [inline ISSUE_TEMPLATE.md]: Now that the template is not so in your face maybe we don't need this anymore?

**Comment by Jguer** [comment]
For some reason my markdown got reformatted

**Pull request merged**